### PR TITLE
Blast: Explorer update and Adding SYN to bridgeMap

### PR DESCRIPTION
**Pull request by Defi-Moses**
This PR 
1. Updates the Synapse constants package to support everything needed for Blast
2. Updates the Explorer to support Blast transactions
3. Updates bridgeMap so that synapse-interface can support SYN bridging. 

## Summary by CodeRabbit

- **New Features**
    - Enhanced Chain Chart with a new `<Bar>` component for improved visualization.
    - Introduced new SVG icons and images (blast, USDB) for a better UI.
    - Added support for the BLAST chain with details and bridge contract address.
    - Updated bridge map with new tokens and destinations for improved bridging.
    - Included new token `USDB` for BLAST chain with detailed information.
    - Added a new RPC provider for better network connectivity.

- **Refactor**
    - Streamlined bridge mapping logic and removed legacy token handling for performance.

- **Chores**
    - Updated `synapse-constants` package version.
    - Added list of unsupported Ethereum addresses.

dd16446c03bd9caff69719ecb43dfce1ff820e7f: [explorer-ui preview link ](https://sanguine-p0mk0edsg-synapsecns.vercel.app)
dd16446c03bd9caff69719ecb43dfce1ff820e7f: [synapse-interface preview link ](https://sanguine-synapse-interface-a8d4it6n3-synapsecns.vercel.app)
20ec0776409e368ae80f961d4fa60bcf62728b94: [synapse-interface preview link ](https://sanguine-synapse-interface-lxpxoig2n-synapsecns.vercel.app)
20ec0776409e368ae80f961d4fa60bcf62728b94: [explorer-ui preview link ](https://sanguine-bjlwuj5vr-synapsecns.vercel.app)
9c3f6e666b2f0923f74afa08b1a57595494ecad7: [synapse-interface preview link ](https://sanguine-synapse-interface-lq3zczpee-synapsecns.vercel.app)
9c3f6e666b2f0923f74afa08b1a57595494ecad7: [explorer-ui preview link ](https://sanguine-evojus1g3-synapsecns.vercel.app)
ff8fd740dce47962d973f4a15d55d550aa697d57: [synapse-interface preview link ](https://sanguine-synapse-interface-7vp1kar1r-synapsecns.vercel.app)
ff8fd740dce47962d973f4a15d55d550aa697d57: [explorer-ui preview link ](https://sanguine-ke121adas-synapsecns.vercel.app)

### CODE REVIEW

File: packages/synapse-constants/constants/tokens/bridgeable.ts [hunk 1]
```diff
@@ -1263,3 +1270,22 @@ export const USDBC = new Token({
...
+  swapExceptions: {},
+  symbol: 'USDbC',
+  name: 'Blast Rebasing USD',
+  logo: usdbLogo,
+  swapableType: 'USD',
+  swapableOn: [],
+  color: 'blue',
+  priorityRank: 125,
+  routeSymbol: 'USDB',
+})
```
[the remark this answers is not in the document]
Comment (Defi-Moses) [reply packages/synapse-constants/constants/tokens/bridgeable.ts:1291]: fixed

File: packages/synapse-constants/constants/tokens/bridgeable.ts [hunk 2]
```diff
@@ -1263,3 +1268,22 @@ export const USDBC = new Token({
...
+    [CHAINS.BLAST.id]: 18,
+  },
+  swapExceptions: {},
+  symbol: 'USDB',
+  name: 'Blast Rebasing USD',
+  logo: usdbLogo,
+  swapableType: 'USD',
+  swapableOn: [],
+  color: 'blue',
+  priorityRank: 125,
```
Comment (ChiTimesChi) [inline packages/synapse-constants/constants/tokens/bridgeable.ts:1287]: `priorityRank` should be 1 here I think

**Review by ChiTimesChi — APPROVED** [review]
LGTM with the exception of USDB's `priorityRank`

File: packages/synapse-constants/constants/tokens/bridgeable.ts (hunk 2 above)
Comment (trajan0x) [reply packages/synapse-constants/constants/tokens/bridgeable.ts:1287]: @Defi-Moses let's do this in next pr so we can hotfix explorer ui

Comment (trajan0x) [reply packages/synapse-constants/constants/tokens/bridgeable.ts:1287]: this won't go live to synapse-interface just yet anyway

**Pull request merged**